entries.py

```diff
@@ -146,6 +162,22 @@
 	pronunciation='"el-woz-ee", derived from Xhosa word for knowledge', 
 	seealso="")
 
+FAIR = GlossEntry("FAIR", 
+	acronym_full="", 
+	definition="A set of guidelines to improve the Findability, Accessibility, Interoperability, and Reuse of digital assets", 
+	furtherreading="https://www.go-fair.org/fair-principles/",
+	institute="", 
+	pronunciation='', 
+	seealso="")
+
+GA4GH = GlossEntry("GA4GH", 
+	acronym_full="Global Alliance For Genomics and Health", 
+	definition="The Global Alliance for Genomics and Health aims to accelerate progress in genomic research and human health by cultivating a common framework of standards and harmonized approaches for effective and responsible genomic and health-related data sharing.", 
```

Comment:
Is there a way to "dictionaryafy" this by making it into a fragment? Would it be correct to call GA4GH "a network of public and private institutions which aims to accelerate..." or perhaps "a coalition of several institutions which aims to accelerate..."

Reply:
I think either of those options is fine.